.github/workflows/releases.yml

```diff
@@ -83,7 +83,7 @@ jobs:
           echo "${{secrets.GPG_KEY}}" | base64 -d | gpg --import --no-tty --batch --yes
           echo "allow-preset-passphrase" > ~/.gnupg/gpg-agent.conf
           gpg-connect-agent RELOADAGENT /bye
-          echo "${{secrets.GPG_PASSPHRASE}}" | /usr/lib/gnupg2/gpg-preset-passphrase --preset 867DAD5051270B843EF54F6186FA10E3A1D22DC5
+          echo "${{secrets.GPG_PASSPHRASE}}" | /usr/lib/gnupg2/gpg-preset-passphrase --preset "${{secrets.GPG_KEYGRIP}}"
```

Comment:
This is good for our purposes, but in general it's best to avoid GitHub Action syntax `${{...}}` inside bash scripts, and instead assign `${{secrets.*}}` values to environment variables and then use those environment variables in the bash script. That approach avoids a whole set of escaping problems when it comes to secret values that may have bash syntax like quotes or `$` in them.

We can fix this as subsequent ship.